extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -116,19 +116,25 @@
 }
 
 .i-amphtml-lbv[gallery-view] .i-amphtml-lbv-gallery {
-  display: flex;
-  flex-wrap: wrap;
-  align-content: flex-start;
+  display: grid;
+  justify-content: center;
+  grid-gap: 5px;
+  grid-template-columns: repeat(3, calc(100vw/3 - 6px));
```

Comment:
let's try to remove dependency to `vw`. Grid-layout should be flexible enough to be based on the "container" width.